src/main/java/com/esri/samples/raster/raster_rendering_rule/README.md

```diff
@@ -0,0 +1,27 @@
+<h1>Raster Rendering Rule</h1>
+
+<p>Create an <code>ImageServiceRaster</code>, fetch the <code>RenderingRules</code> from the service info, and use a <code>RenderingRule</code> to create an <code>ImageServiceRaster</code> and add it to a raster layer. </p>
+
+<p><img src="RasterRenderingRule.png"/></p>
+
+<h2>How to use the sample</h2>
+
+<p>Run the sample and use the drop-down menu at the top to select a rendering rule.</p>
+
+<h2>How it works</h2>
+
```

Comment:
Ordered list, including methods specific to Java where appropriate. e.g. this is a good one to mention when describing getting the rendering rule info from the image service raster.
`imageServiceRaster.getServiceInfo().getRenderingRuleInfos();`